token/services/network/fabric/endorsement.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package fabric
+
+import (
+	"github.com/hyperledger-labs/fabric-smart-client/platform/fabric"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/fabric/services/chaincode"
+	view2 "github.com/hyperledger-labs/fabric-smart-client/platform/view"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
+	token2 "github.com/hyperledger-labs/fabric-token-sdk/token"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/network/driver"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/network/fabric/fts"
+	"github.com/pkg/errors"
+)
+
+type Endorsement interface {
+	Endorse(context view.Context, tms *token2.ManagementService, requestRaw []byte, signer view.Identity, txID driver.TxID) (driver.Envelope, error)
+}
+
+type FSCEndorsement struct {
+	Endorsers []view.Identity
+}
+
+func (e *FSCEndorsement) Endorse(context view.Context, tms *token2.ManagementService, requestRaw []byte, signer view.Identity, txID driver.TxID) (driver.Envelope, error) {
```

Comment:
We could make this a service and inject tms and viewmanager

Reply:
This has the same signature of the method in the Network interface. We need tms at the time of the invocation. The viewManager con be injected, sure.